lib/file-template.js

```diff
@@ -1,12 +1,13 @@
 'use strict'
 
-const fileTemplate=(language, version) => {
-	var config = `language: ${language}\n${language}:\n`
-	for (let i = 0; i < version.length; i++) {
-		config += `  - "${version[i]}"\n`
-	}
+const fileTemplate = (language, ...version) => {
```

Comment:
Awesome! One suggestion tho... if you agree, what to hear your thoughts on it 😅 

You are using `...version` which is okay and works fine. But I do believe that it should receive an `array` instead. So we can do like this:

```js
const fileTemplate = (language, versions) => {
  let config = `language: ${language}\n${language}:\n`
  versions.map(v => {
    config += `  - ${v}\n`
    return v
  })

  return config
}
```

And to pass the versions it's easier:
```js
const versions = ['5', '6', 'iojs']

fileTemplate('node_js', versions)
```

Against having to pass like:
```js
fileTemplate('node_js', '5', '6', 'iojs')
```

Ps: By doing that it will break two tests `./__tests__/file-template.js` and `index.js`. You only have to pass an array there to fix it.